src/components/InvertedFlatList/BaseInvertedFlatList.js

```diff
@@ -149,7 +145,6 @@ class BaseInvertedFlatList extends Component {
                 // We keep this property very low so that chat switching remains fast
                 maxToRenderPerBatch={1}
                 windowSize={15}
-                removeClippedSubviews={this.props.shouldRemoveClippedSubviews}
```

Comment:
Hard-code this to `false` because the default is `true` on Android: https://reactnative.dev/docs/flatlist#removeclippedsubviews